brave/brave-core
# Fixes new year calculation

Resolves https://github.com/brave/brave-browser/issues/2882

## Submitter Checklist:

- Verified that these changes build without errors on
- Verified that these changes pass automated tests (`npm test brave_unit_tests && npm test brave_browser_tests`) on

## Test Plan:
- create profile on an old version where entries are from December
- start browser in this year and add one site (that site should have 100% in ac table)
- start browser with this branch
- visit that site again
- go to settings page and make sure that table is ok again

## Reviewer Checklist:
